CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ request related to the change, then we may provide the commit.
 
 This is not a comprehensive list of changes but rather a hand-curated collection of the more notable ones. For a comprehensive history, see the [OpenSim Core GitHub repo](https://github.com/opensim-org/opensim-core).
 
+v4.5
+====
+- Added `AbstractPath` which is a base class for `GeometryPath` and other path types (#3388). TODO: add API changes.
```

Comment:
Per your suggestions, I'm opting to revert API changes since supporting `getGeometryPath` and `updGeometryPath` is straight-forward anyway. But I agree, in the end, I'd prefer minimal API changes for those reasons.